packages/components/atoms/f-button/test/component/f-button.component.spec.js

```diff
@@ -1,74 +1,57 @@
-const ActionButton = require('../../test-utils/component-objects/f-button--action.component');
-const LinkButton = require('../../test-utils/component-objects/f-button--link.component');
-const IconButton = require('../../test-utils/component-objects/f-button--icon.component');
-
-let button;
+import ActionButton from '../../test-utils/component-objects/f-button--action.component';
+import LinkButton from '../../test-utils/component-objects/f-button--link.component';
+import IconButton from '../../test-utils/component-objects/f-button--icon.component';
 
 describe('Action f-button component tests', () => {
-    beforeEach(() => {
-        // Arrange
-        button = new ActionButton();
-    });
-
     it('should display the f-button action component', async () => {
         // Act
-        await button.load();
+        await ActionButton.load();
 
         // Assert
-        await expect(await button.isComponentDisplayed()).toBe(true);
+        await expect(await ActionButton.isComponentDisplayed()).toBe(true);
```

Comment:
Tiny NIT: I'd personally probably store `await ActionButton.isComponentDisplayed()` as a variable just for readability. Entirely subjective of course and not a change request :)